frontend/alert/src/Timeline.js

```diff
@@ -0,0 +1,222 @@
+import React, {useEffect} from 'react';
+import styled from "styled-components";
+
+const TimelineContainer = styled.div`
+  position: absolute;
+  right: ${props=>props.offScreen===true ? "-300px" : "0px"};
+  top: 0;
+  width: 250px;
+  height:100vh;
+  padding: 20px;
+  box-shadow: 0 2px 9px 0 rgba(0, 0, 0, 0.08);
+  background: white;
+  transition:0.5s all;
+  z-index: 100;
+`
+const ScrollContainer = styled.div`
+  overflow-y: scroll;
+  height:calc(100vh - 200px);
+`
+
+const MyButton = styled.button`
+  outline: none;
+  border: none;
+  background: none;
+  position: absolute;
+  right: 20px;
+  top: 20px;
+  font-size: 15px;
+  i{
+    color: #9d9d9d;
+    transition: 0.2s;
+    :hover{
+      color: #5a5a5a;
+    }
+  }
+`
+
+const Heading = styled.h3`
+  font-size: 25px;
+  color: #282828;
+  margin-bottom: 0px;
+  padding-bottom: 0px;
+`;
+
+const Subheading = styled.h5`
+  font-size: 18px;
+  color: #282828;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  font-weight: normal;
+`;
+
+const Center = styled.div`
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+`
+
+
+
+const ClosedBadge = styled.div`
+  padding: 5px 10px;
+  height: 20px;
+  border-radius: 3px;
+  font-weight: 600;
+  color: #e8746a;
+  background: #f9dcda;
+`
+
+const OpenBadge = styled(ClosedBadge)`
+  color:
+`
+
+const Segment = styled.div`
+  width: 2px;
+  height: ${props=>props.height + 5}px;
+  background: ${props=>props.type=="opened" ? "#78d381" : "#cbd5dd"};
+`
+const MyCircle = styled.div`
+  color: ${props=>props.type=="opened" ? "#78d381" : "#cbd5dd"};
+`
+
+const TimeStyle = styled.div`
+  position: absolute;
+  top: ${props => props.offset}px;
+  right: 50px;
+`
+
+const DateStyle = styled.div`
+  position: absolute;
+  top: ${props => props.offset}px;
+  left: 10px;
+`
+
+const CourseIndicator = ({time, type, offset}) => {
+  let convertedTime = convertTime(time);
+  return <TimeStyle offset={offset}>{convertedTime[1]}</TimeStyle>
+}
+
+const FlexRow = styled.div`
+  display: flex;
+  flex-direction: row;
+  position: relative;
+`
+
+const LeftRight = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+`
+
+function convertTime(timeString){
```

Comment:
```suggestion
const convertTime = timeString => {
```

Could all of these use the new function syntax?